.travis.yml

```diff
@@ -8,6 +8,9 @@ script:
 matrix:
   include:
   - jdk: oraclejdk8
+    after_success:
+      - mvn cobertura:cobertura
+      - bash <(curl -s https://codecov.io/bash)
```

Comment:
Have you considered just integrating this into the maven lifecycle? Keeping things wrapped into the maven lifecycle increases portability and makes things like #1012 much easier.

Reply:
No idea how to use maven, unfortunately :/

Reply:
I'm not sure that it's worth it:

- the .travis-ci.yml and .gitlab-ci.yml are simple enough
- everyone knows yaml, fewer know maven lifecycles
- it's unlikely that we'll want to send the coverage from many places: it'll only be from travis-ci, and should we move entirely to gitlab-ci, it'll just be 3 lines to move to an other yaml file
- having this in the yaml files is the recommended way to use codecov.